src/preferences/effectmanifesttablemodel.cpp

```diff
@@ -146,8 +146,17 @@ Qt::DropActions EffectManifestTableModel::supportedDropActions() const {
 }
 
 bool EffectManifestTableModel::removeRows(int row, int count, const QModelIndex& parent) {
-    Q_UNUSED(parent);
-    beginRemoveRows(QModelIndex(), row, row + count - 1);
+    if (!count) {
+        // nothing to do
+        return true;
+    }
+    VERIFY_OR_DEBUG_ASSERT(count > 0 && row < m_manifests.size() &&
+            row + count <= m_manifests.size()) {
```

Comment:
The second condition is already being enforced by the result of the first and the third one. I'd rather check we're not being passed some negative row index.
```suggestion
    VERIFY_OR_DEBUG_ASSERT(count > 0 && row >= 0 &&
            row + count <= m_manifests.size()) {
```